core/src/main/scala/org/apache/spark/storage/BlockManagerMasterEndpoint.scala

```diff
@@ -325,6 +328,10 @@ class BlockManagerMasterEndpoint(
     }.toMap
   }
 
+  private def isExecutorAlive(executorId: String): Boolean = {
+    blockManagerIdByExecutor.contains(executorId)
```

Comment:
This is just a one-line method, can we inline it?

Reply:
> This is just a one-line method, can we inline it?
Done.
@cloud-fan  Thanks for your valuable advices, I think there are no other problems.